Momento/MomentoSdk.csproj

```diff
@@ -1,6 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <TargetFramework>netstandard1.2</TargetFramework>

```

Comment:
Not sure how this comes into play. My most immediate concern is how can we test this with specific version. Similar to how we do for python - we test with different python versions.